maintenance_sign_oca/demo/sign_oca_role.xml

```diff
@@ -1,7 +1,7 @@
 <odoo>
     <record id="role_maintenance_equipment_owner" model="sign.oca.role">
         <field name="name">Equipment owner</field>
-        <field name="partner_type">expression</field>
+        <field name="partner_selection_policy">expression</field>
```

Comment:
This highlights a migration script needed on the base module, isn't it?

Reply:
Exactly, already added https://github.com/OCA/sign/blob/16.0/sign_oca/migrations/16.0.1.1.1/pre-migration.py